blocks/library/paragraph/style.scss

```diff
@@ -18,7 +18,3 @@ p.has-drop-cap:not( :focus )  {
 p.has-background {
 	padding: 20px 30px;
 }
-
-.editor-block-list__block:not( .is-multi-selected ) .wp-block-paragraph {
-	background: white;
-}
```

Comment:
Why remove this?

Reply:
Because the `.editor-block-list__block` is an editor class and shouldn't be used in the `style.scss` which is loaded in the frontend as well. The same code is also in `editor.style.scss`